watertap/unit_models/translators/tests/test_translator_asm1_adm1.py

```diff
@@ -0,0 +1,402 @@
+        assert (
+            abs(
+                value(
+                    asmadm.fs.unit.CODs[0]
+                    - (
+                        asmadm.fs.unit.outlet.conc_mass_comp[0, "S_I"]
+                        + asmadm.fs.unit.outlet.conc_mass_comp[0, "S_su"]
+                        + asmadm.fs.unit.outlet.conc_mass_comp[0, "S_aa"]
+                        + asmadm.fs.unit.outlet.conc_mass_comp[0, "S_fa"]
+                        + asmadm.fs.unit.outlet.conc_mass_comp[0, "S_va"]
+                        + asmadm.fs.unit.outlet.conc_mass_comp[0, "S_bu"]
+                        + asmadm.fs.unit.outlet.conc_mass_comp[0, "S_pro"]
+                        + asmadm.fs.unit.outlet.conc_mass_comp[0, "S_ac"]
+                        + asmadm.fs.unit.outlet.conc_mass_comp[0, "S_h2"]
+                        + asmadm.fs.unit.outlet.conc_mass_comp[0, "S_ch4"]
+                    )
+                )
+            )
+            <= 1e-5
+        )
+
+        assert (
+            abs(
+                value(
+                    asmadm.fs.unit.CODp[0]
+                    - (
+                        asmadm.fs.unit.outlet.conc_mass_comp[0, "X_I"]
+                        + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_su"]
+                        + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_aa"]
+                        + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_fa"]
+                        + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_c4"]
+                        + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_pro"]
+                        + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_h2"]
+                        + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_c"]
+                        + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_ch"]
+                        + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_pr"]
+                        + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_li"]
+                    )
+                )
+            )
+            <= 1e-5
+        )
+
+        # assert (
+        #     abs(
+        #         value(
+        #             asmadm.fs.unit.TKN[0]
+        #             - (asmadm.fs.unit.outlet.conc_mass_comp[0, "S_IN"]
+        #             + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_su"]
+        #             + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_aa"]
+        #             + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_fa"]
+        #             + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_c4"]
+        #             + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_pro"]
+        #             + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_h2"]
+        #             + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_c"]
+        #             + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_ch"]
+        #             + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_pr"]
+        #             + asmadm.fs.unit.outlet.conc_mass_comp[0, "X_li"])
+        #         )
+        #     )
+        #     <= 1e-6
+        # )
```

Comment:
Is this bit passing? Are there more tests you should add? It looks that way from glancing at the unit